src/main/kotlin/no/nav/syfo/api/endpoints/ManglendeMedvirkningEndpoints.kt

```diff
@@ -0,0 +1,61 @@
+package no.nav.syfo.api.endpoints
+
+import io.ktor.http.HttpStatusCode
+import io.ktor.server.application.call
+import io.ktor.server.request.receive
+import io.ktor.server.response.respond
+import io.ktor.server.routing.Route
+import io.ktor.server.routing.get
+import io.ktor.server.routing.post
+import io.ktor.server.routing.route
+import no.nav.syfo.api.model.NewVurderingRequestDTO
+import no.nav.syfo.api.model.NewVurderingResponseDTO
+import no.nav.syfo.application.VurderingService
+import no.nav.syfo.infrastructure.clients.veiledertilgang.VeilederTilgangskontrollClient
+import no.nav.syfo.infrastructure.clients.veiledertilgang.validateVeilederAccess
+import no.nav.syfo.util.getCallId
+import no.nav.syfo.util.getNAVIdent
+
+fun Route.registerManglendeMedvirkningEndpoints(
+    veilederTilgangskontrollClient: VeilederTilgangskontrollClient,
+    vurderingService: VurderingService,
+) {
+    route("/api/internad/v1/manglende-medvirkning") {
+        get("/vurderinger") {
+            // TODO: Implement
+            call.respond(HttpStatusCode.NotImplemented)
+        }
+
+        post("/vurderinger") {
+            val requestDTO = call.receive<NewVurderingRequestDTO>()
+
+            validateVeilederAccess(
+                action = "POST /vurderinger",
+                personident = requestDTO.personident,
+                veilederTilgangskontrollClient = veilederTilgangskontrollClient,
+            ) {
+                val callId = call.getCallId()
+                if (requestDTO.document.isEmpty()) {
+                    throw IllegalArgumentException("Vurdering can't have empty document, callId: $callId")
+                }
+
```

Comment:
Burde være en sjekk her på om det allerede er opprettet en vurdering? (dvs unngå duplikater)

Reply:
Hvordan kan vi gjøre det på en smart måte? 🤔 Hente ut fra database og se om siste vurdering på person er samme vurdering? 
Hvordan kan vi gjøre det så lite omstendelig som mulig?

Reply:
Ja, det var noe sånt jeg så for meg. Men det forutsetter jo at vurderingene blir "lukket" på en måte når de er ferdig behandlet.

Kan kanskje ordnes i en egen PR (dvs lage en lapp på det=?